packages/@aws-cdk/aws-servicecatalog/lib/constraints.ts

```diff
@@ -0,0 +1,29 @@
+import { AcceptLanguage } from './common';
+
+/**
+ * Properties for governance mechanisms and constraints.
+ */
+export interface ConstraintProps {
+  /**
+   * The language code.
```

Comment:
My understanding is that the largest use case for accept language is just the language used by error handling.  Customers might have a multi-region and different local language for their resources so they can specify the accept language as needed.

Reply:
In any case I will add  that to the documentation here

Reply:
So, in the case of the tag update constraint, the error message that a customer of that "Product in Portfolio" would get if they tried to set the tags in their instance of the Product would be in a different language depending on what was set here?

Reply:
if they are doing some downstream operation in the console or cli and it violated a constraint then yes the surfaced error should be controlled by what is set here.

Reply:
Ok, got it. I would probably call it something like `errorMessageLanguage` or similar then, because `acceptLanguage` here just gives me 0 information on what is this actually used for.

Reply:
upon review further we have the accept language is used for both status updates and error messages.  To try to keep it connected I think using `acceptedMessageLanguage` conveys the meaning, and added to doc strings.